build/Microsoft.DotNet.Cli.BundledSdks.proj

```diff
@@ -0,0 +1,46 @@
+<Project ToolsVersion="15.0" DefaultTargets="CopySdkToOutput">
+  <!-- workaround for https://github.com/Microsoft/msbuild/issues/885 -->
+  <!-- renaming the property because the original property is a global property and therefore
+       cannot be redefined at runtime. -->
+  <PropertyGroup>
+    <CLIBuildDllPath>$([MSBuild]::Unescape($(CLIBuildDll)))</CLIBuildDllPath>
+  </PropertyGroup>
+
+  <UsingTask TaskName="DotNetRestore" AssemblyFile="$(CLIBuildDllPath)" />
+
+  <Target Name="CopySdkToOutput"
+          DependsOnTargets="PrepareBundledSdksProps;
+                            EnsureSdkRestored;
+                            GetSdkItemsToCopy"
+          Inputs="@(SdkContent)"
+          Outputs="@(SdkContent->'$(oz)/$(SdkPackageName)/$(SdkPackageVersion)/%(RecursiveDir)%(FileName)%(Extension)')">
+    <Copy SourceFiles="@(SdkContent)"
+          DestinationFiles="@(SdkContent->'$(SdkLayoutDirectory)/%(RecursiveDir)%(FileName)%(Extension)')" />
+  </Target>
+
+  <Target Name="GetSdkItemsToCopy">
+    <ItemGroup>
+      <SdkContent Include="$(SdkNuPkgPath)/**/*" 
+                  Exclude="$(SdkNuPkgPath)/$(SdkPackageName).nuspec;
+                           $(SdkNuPkgPath)/$(SdkPackageName).$(SdkPackageVersion).nupkg;
+                           $(SdkNuPkgPath)/$(SdkPackageName).$(SdkPackageVersion).nupkg.sha512" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="EnsureSdkRestored"
+          Condition="!Exists('$(SdkNuPkgPath)/$(SdkPackageName.ToLower()).nuspec')">
+    
+    <Message Text="$(SdkNuPkgPath)/$(SdkPackageName.ToLower()).nuspec" 
```

Comment:
It seems like this message should say something like "Restoring <PackageName> <Package Version>" instead of just the path to the .nuspec.